packages/form-fields/partials/label/index.tsx

```diff
@@ -0,0 +1,55 @@
+import s from './style.module.css'
+import clsx from 'clsx'
+import { IconAlertTriangleFill16 } from '@hashicorp/flight-icons/svg-react/alert-triangle-fill-16'
+
+interface LabelProps {
+  label: string
+  helpText?: string
+  helpId?: string
+  inputId?: string
```

Comment:
💭 How do you feel about naming this `htmlFor`, for parity with the native `label` (or at least native JSX version)?

It's looks like a 1:1 _passthrough_ so it seems like there's no harm in doing that.

Reply:
Sounds good to me! Let's do it.